block-non-tracking-third-party-cookies/README.md

```diff
@@ -0,0 +1,86 @@
+# Third Party Cookies Bocking Tests
+
+Privacy Feature: https://app.asana.com/0/1198207348643509/1199093921854081/f
+
+## Goals
+
+This set of tests verifies implementation of third party non-tracker cookie blocking. In particular it focuses on verifying that:
+
+- `cookie` and `set-cookie` headers are being removed from requests to 3p non-trackers
+- multiple `cookie` headers are handled
+- remote config exceptions are taken into account
+- `document.cookie` disallows setting / reading cookies in non-tracking contexts
```

Comment:
we also need to account for same-entity rule that we decided to keep in extensions